README.md

```diff
@@ -182,7 +182,7 @@ Unity Catalog can be built using [sbt](https://www.scala-sbt.org/).
 
 To build UC (incl. [Spark Integration](./connectors/spark) module), run the following command:
 
-```shell
```

Comment:
@HorizonNet in the _past_, using `shell` vs. `bash` had slightly different outputs for readability.  Saying this, I have not checked this since.  I wanted to confirm with you that there really is no difference and then yes, switching to `sh` would certainly simplify things.

Reply:
@dennyglee I did a comparison of the rendering by using this branch and what we have online right now. They look the same. I also looked into the Material theme for MkDocs, which we are using. It uses Pygments, where `bash`, `sh` and `shell` are all resolved to the [same lexer](https://pygments.org/languages/).

Reply:
Brilliant, thanks for confirming @HorizonNet !